docs/content/docs/tasks/restart-application-deployment/_index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Restart an Application Deployment
+description: Learn how to restart an unsuccessful Keptn Application Deployment.
+icon: concepts
+layout: quickstart
+weight: 20
+hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+## Restart an Application Deployment
+
+During the deployment of a `KeptnApp`, it might be that the deployment fails due to an unsuccessful pre-deployment evaluation or pre-deployment check.
```

Comment:
```suggestion
During the deployment of a `KeptnApp`, it might be that the deployment fails due to an unsuccessful pre-deployment evaluation or pre-deployment task.
```